src/main/resources/db/changelog/changes/12-create-order-table.yaml

```diff
@@ -0,0 +1,56 @@
+databaseChangeLog:
+  - changeSet:
+      id: 12
```

Comment:
I believe something meaninfull should be here

Reply:
All previous changeSets had just id.
I will change all of them after merging everything in master